modules/object_storage/main.tf

```diff
@@ -3,11 +3,11 @@ resource "random_pet" "gcs" {
 }
 
 resource "google_storage_bucket" "tfe" {
-  name     = "${var.namespace}-storage-${random_pet.gcs.id}"
-  location = "us"
-
-  force_destroy = true
-  labels        = var.labels
+  name                        = "${var.namespace}-storage-${random_pet.gcs.id}"
+  location                    = "us"
+  uniform_bucket_level_access = true
```

Comment:
This is a new request by security that technically supports our usage below this resource but makes that position explicit.  We could pass this option forward to the root module variables or we can be opinionated as this current change suggests.  I'm open to either approach.

Reply:
I think hard-coding is fine 🎉